app/helpers/simple_form_helper.rb

```diff
@@ -79,7 +79,7 @@ def inputs_for(model, form_builder)
       if val
         res << form_builder.hidden_field(model.class.reflections[col].foreign_key, value: val.id)
       else
-        res << form_builder.association(col, label: "#{model.class.name.underscore}.#{col}".tmf)
+        res << form_builder.association(col, label: "#{model.class.table_name.singularize}.#{col}".tmf)
```

Comment:
Why don't we add the corresponding methods on the ActiveRecord::Base class, instead of a big copy/paste everytime ?